install.sh

```diff
@@ -18,32 +18,37 @@ guess_system_package_manager(){
         SYSTEM_PACKAGE_TYPE="rpm"
         SYSTEM_PACKAGE_MANAGER_INSTALL="dnf -y install"
         SYSTEM_PACKAGE_MANAGER_UPDATE="dnf --refresh check-update"
+        PIP_INSTALL_CMD="pip install "
```

Comment:
Move this variable definition into the `Variables` section so we don't have to redefine it each time we add support for a new package manager.